eZ/Publish/Core/QueryType/QueryParameterContentViewQueryTypeMapper.php

```diff
@@ -61,13 +61,13 @@ private function extractParameters(ContentView $contentView)
 
     /**
      * @param ContentView $contentView
-     * @param string $queryParameterValue
+     * @param string|string[] $queryParameterValue
      *
      * @return mixed
      */
     private function evaluateExpression(ContentView $contentView, $queryParameterValue)
     {
-        if (substr($queryParameterValue, 0, 2) === '@=') {
+        if (is_string($queryParameterValue) && substr($queryParameterValue, 0, 2) === '@=') {
```

Comment:
That would be nice enhancement. But maybe a follow up? _(I'm assuming this fix is needed for demo bundle as we speak)_

Besides there might be some use cases in being able to pass in objects as param as well here.